src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/ViewBacklogPage.kt

```diff
@@ -38,4 +40,8 @@ class ViewBacklogPage(
             textToBePresentInElementLocated(By.cssSelector(".ghx-backlog-container .ghx-name"), expectedSprintName)
         )
     }
+
+    fun closeAuiFlags() {
+        popUps.waitUntilAuiFlagsAreGone()
```

Comment:
The name of this function suggests that it only waits until flags are hidden, but in fact, it also closes them. I'm pasting a snippet for reference:
```   
fun waitUntilAuiFlagsAreGone(): NotificationPopUps {
        try {
            waitUntilAuiFlagsAreGone(Duration.ofSeconds(5))
            return this
        } catch (e: TimeoutException) {
        }

        try {
            dismissAuiFlags()
            waitUntilAuiFlagsAreGone(Duration.ofSeconds(30))
            return this
        } catch (e: Exception) {
            throw e
        }
    }
```

Reply:
Indeed a bit confusing

Reply:
Good job though of extracting this wait to a method that explains the actual action done. It's the best way to do that on this level.